frontend/packages/shared/human.ts

```diff
@@ -12,29 +12,29 @@ export function humanBalance(balance: string | number | BigNumber) {
   const VALUES = [
     {
       unit: "K",
-      size: THOUSAND
+      size: THOUSAND,
```

Comment:
just guessing, may be there is existing nmp for all stuff currency?

Reply:
Did not wanted to introduce a new package just for this. But if you know something I would be glad to incorporate that.